packages/web-components/src/components/tabs-extended/tabs-extended.ts

```diff
@@ -166,25 +181,28 @@ class DDSTabsExtended extends StableSelectorMixin(LitElement) {
     this._isLTR = window.getComputedStyle(this).direction === 'ltr';
     this._activeTabIndex = parseInt(this._activeTab, 10);
 
-    this._tabItems.map((tab, index) => {
-      (tab as DDSTab).selected = index === this._activeTabIndex;
-      (tab as DDSTab).setIndex(index);
-      const navLink = this.shadowRoot!.querySelectorAll(
-        `.${prefix}--tabs__nav-link`
-      )[index];
-      const navText = navLink!.querySelector('div p');
-      if (navText!.scrollHeight > navText!.clientHeight) {
-        const label = (tab as DDSTab).getAttribute('label');
-        if (label) {
-          navLink!.setAttribute('aria-label', label);
-          navLink!.setAttribute('hasTooltip', label);
+    // Set aria-label on tabs for desktop.
+    if (!this._isMobileVersion) {
+      this._tabItems.forEach((tab, index) => {
+        (tab as DDSTab).selected = index === this._activeTabIndex;
+        (tab as DDSTab).setIndex(index);
+        const navLink = this.shadowRoot!.querySelectorAll(
+          `.${prefix}--tabs__nav-link`
+        )[index];
+        const navText = navLink!.querySelector('div p');
+        if (navText!.scrollHeight > navText!.clientHeight) {
+          const label = (tab as DDSTab).getAttribute('label');
+          if (label) {
+            navLink!.setAttribute('aria-label', label);
+            navLink!.setAttribute('hasTooltip', label);
+          }
         }
-      }
-      return tab;
-    });
+        return tab;
```

Comment:
Small suggestion to remove the `return` statement, since the return from the callback of a `forEach` is discarded anyway (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach).

```suggestion
```